docs/proposals/scheduler/scheduler.proto

```diff
@@ -0,0 +1,81 @@
+syntax = "proto3";
+
+package scheduler;
+import "google/protobuf/empty.proto";
+
+enum Status {
+    RUNNING = 0;
+    STARTING = 1;
+    STOPPED = 2;
+    STOPPING = 3;
+    DESTROYING = 4;
+    TERMINATED = 5;
+    CRASHED = 6;
+    FAILED = 7;
+    SCHEDULING = 8;
+    SCHEDULED = 9;
+}
+
+enum Type {
+    CONTAINER = 0;
+}
+
+message Instance {
+    string id = 1;
+    string name = 2;
+    Type type = 3;
+    Status status = 4;
+    string uri = 5;
+    repeated string environnement = 6;
+    Resource resource = 7;
+    repeated string ports = 8;
+    string ip = 9;
+}
+
+message ResourceSummary {
+    int32 cpu = 1;
+    int32 memory = 2;
+    int32 disk = 3;
+}
+
+message Resource {
+    ResourceSummary max = 1;
+    ResourceSummary usage = 2;
+}
+
+message InstanceStatus {
+    string id = 1;
+    Status status = 2;
+    string description = 3;
+}
+
+message NodeStatus {
+    string id = 1;
+    Status status = 2;
+    string description = 3;
+    Resource resource = 4;
+    repeated Instance instances = 5;
+}
+
+message NodeRegisterRequest {
+    string certificate = 1;
+}
+
+message NodeRegisterResponse {
+    int32 code = 1;
+    string description = 2;
+    string subnet = 3;
```

Comment:
I believe you mean a CIDR (https://en.wikipedia.org/wiki/Classless_Inter-Domain_Routing) here.

Reply:
Yes